src/release/backporting.md

```diff
@@ -5,36 +5,47 @@ Only a few people are even aware of the process, but this is actually something
-the beta branch. Those cherry-picks are then submitted as a PR _against the
-beta branch_, with a title started with `[beta]` (so reviewers can see its
-specialness). The OP of that PR should contain links to all the PRs being
-backported. [Here's an example](https://github.com/rust-lang/rust/pull/64097).
-Anybody can make these PRs!
-
-After that a reviewer needs to verify that the backport looks correct, that it's
-submitted to the beta branch, and then approve via `@bors: r+`. Finally, they
-need to follow the links to the original PRs and _remove the `beta-nominated`
-tag_ (people forget to do this a lot). This last step indicates that the
-backport has been completed, so the `beta-nominated` and `beta-accepted` tags
-have three states.
-
-If, on the other hand, a backport is declined the `beta-nominated` label is
-removed, closing the procedure.
+Backports of PRs to the beta branch are usually only done to fix regressions.
+Getting a PR backported to the beta branch involves the following process:
+
+1. Add the label [`beta-nominated`](https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3Abeta-nominated+-label%3Abeta-accepted) to the PR to be backported.
+   This marks the PR as in the state that it needs attention from the appropriate team to decide if it should be backported.
+   Anybody with triage access is free to add this label.
+
+2. If the team thinks it should be backported, then they should add the [`beta-accepted`](https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3Abeta-accepted) label.
+   Otherwise they should remove the nominated label.
+
+3. Occasionally someone will make a beta rollup PR.
+   This is often done by the release team, but it can be done by anyone.
+   The process here is:
+
+   1. Create a local branch off the `beta` branch.
+   2. Cherry-pick all of the PRs that have both [`beta-nominated` and `beta-accepted`][nominated-plus-accepted] labels.
+      It is usually preferred to not include PRs that have not been merged in case there are any last minute changes, or it fails when running the full CI tests.
+   3. (Recommended) Run some tests locally.
+      It is not uncommon that the backports may not apply cleanly, or the UI tests need to be re-blessed if there are differences in the output.
+   4. Open a PR **against the beta branch** with a title that starts with `[beta]` (so reviewers can see its specialness).
+   5. List all of the PRs being backported in the PR description.
+      [Here's an example](https://github.com/rust-lang/rust/pull/64097).
+
+4. A reviewer (typically from the release team) needs to verify that the backport looks correct and that it's submitted to the beta branch.
+   They will then approve with `@bors r+ rollup=never` (to avoid it being rolled up on accident).
+   If the author of the PR has r+ rights, and has not made significant changes while backporting, they can also self-approve the PR.
+
+5. Go through all of the PRs being backported and:
```

Comment:
This should typically be done as part of making the beta rollup PR (ideally, it'd be done just after merge, but that's annoying to remember to do, so I recommend doing it as you cherry-pick).

Reply:
Sure, I went ahead and moved that step up to the rollup steps.

>  r=me with that amended in some way, or ping me for another review.

I don't have merge permissions here, nor is bors around I think.